cmd/deploy/deploy.go

```diff
@@ -168,7 +150,6 @@ func main() {
 		chainConfig.ChainID.Uint64(): {
 			ChainName:       *l2ChainName,
```

Comment:
Sorry this was missed in previous PR, but we need to set ChainId as well in ChainInfo here.
But definitely can be done in a separate PR.